ext/AvizExt/AvizExt.jl

```diff
@@ -69,19 +66,19 @@ function main_menu()
 
     logo = image(
         f[1, 1],
-        rotr90(load(convert(String, LOGO))),
+        rotr90(load(convert(String, LOGO)));
```

Comment:
comma converted to semicolon?

Reply:
Is there a named argument afterwards? If so, that's why

Reply:
yes, there is, it just looks funny to me when it does `foo(; x = 1)` but I'm not seeing any option to omit the leading semicolon if all args are kwargs.

Reply:
It'll be a syntax error without the semi-colon, won't it? At least I thought it would be...